History.md

```diff
@@ -1,3 +1,13 @@
+1.14 / 2015-03-05
+=================
+
+ * Allow download of files with specified extension for Tinderbox builds on Windows (#264)
+ * Replace --directoy option with --destination to allow a filename for the target file (#92)
```

Comment:
That's just the output from `git changelog --list` so the typing failure happened to me when I commited the changes. I can fix it for the release notes.